system-addon/lib/TopSitesFeed.jsm

```diff
@@ -374,6 +374,10 @@ this.TopSitesFeed = class TopSitesFeed {
   onAction(action) {
     switch (action.type) {
       case at.INIT:
+        // If the feed was previously disabled PREFS_INITIAL_VALUES was never received
+        if (!DEFAULT_TOP_SITES.length) {
```

Comment:
We probably don't want this check? If the pref changed while the feed was off, we would want to replace the contents too